domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimDocument.java

```diff
@@ -9,6 +9,7 @@
 @Getter
 public class ClaimDocument extends CollectionId {
     private final URI documentManagementUrl;
+    private final URI documentManagementBinaryUrl;
```

Comment:
do we need both of these urls? if yes, we should have same in `CCDClaimDocument` as well

Reply:
Ignore this, as it's part of `documentLink` property in CCD side.